code/game/area/Space Station 13 areas.dm

```diff
@@ -164,6 +165,7 @@ var/list/teleportlocs = list()
 	icon_state = "centcom"
 	requires_power = 0
 	has_gravity = 1
+	blob_legit = 0 //Should go without saying, no blobs should take over centcom as a win condition.

```

Comment:
This is a child of space and so it wouldn't need this, it would inherit it from space